app/src/main/cpp/skyline/loader/format.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+namespace skyline::loader {
+    /**
+      * @brief This enumerates the types of ROM files
+      * @note This needs to be synchronized with emu.skyline.loader.BaseLoader.RomFormat
+      */
+    enum class RomFormat {
+        NRO, //!< The NRO format: https://switchbrew.org/wiki/NRO
+        XCI, //!< The XCI format: https://switchbrew.org/wiki/XCI
+        NSP, //!< The NSP format from "nspwn" exploit: https://switchbrew.org/wiki/Switch_System_Flaws
+    };
+}
```

Comment:
Can this be relegated to `loader.h`, I think it'd be fairly fitting in there. Thoughts ?